influx/client.go

```diff
@@ -1,16 +1,198 @@
+// Copyright 2021 InfluxData, Inc. All rights reserved.
+// Use of this source code is governed by MIT
+// license that can be found in the LICENSE file.
+
+// Package influx provides client for InfluxDB server.
 package influx
 
-import "github.com/bonitoo-io/influxdb3-go/influx/configs"
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"mime"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
+	"strings"
+)
+
+// Params holds the parameters for creating a new client.
+// The only mandatory field is ServerURL. AuthToken is also important
+// if authentication was not done outside this client.
+type Params struct {
```

Comment:
Please rename to Configs:

```suggestion
type Configs struct {
```